F5 BIGIP controller -CIS 2.16.1 - errors during pushing data

#### Setup Details
 CIS Version : 2.16.1

 Build: f5networks/k8s-bigip-ctlr:2.16.1
 BIGIP Version: BIG-IP 14.1.4.6 Build 0.0.8 Point Release 6
 AS3 Version: 3.24.0   (f5-appsvcs)   
 Agent Mode: AS3
 Orchestration: OpenShift
 Pool Mode: Cluster
 
#### Description  
  
When the F5 BIGIP controller is pushing changes to the F5 we see regularly the following ERROR logging:

The first two can indicate some performance issue on the F5 BIGIP, but to be confirmed.

2024/05/23 06:31:06 [ERROR] [2024-05-23 06:31:06,498 f5_cccl.bigip ERROR] F5 SDK Error: HTTPSConnectionPool(host='150.45.88.251', port=443): Read timed out. (read timeout=30)
2024/05/23 06:31:06 [ERROR] [2024-05-23 06:31:06,498 __main__ ERROR] CCCL Error: The requested partition OpenShift-ocp-behoi-01 was not found

2024/05/23 06:32:43 [ERROR] [2024-05-23 06:32:43,760 f5_cccl.bigip ERROR] F5 SDK Error: 503 Unexpected Error: Service Unavailable for uri: https://150.45.88.251:443/mgmt/tm/auth/partition/OpenShift-ocp-behoi-01 Text: '{"code":503,"message":"There is an active asynchronous task executing.","errorStack":[],"apiError":32964609}'

These lines, no clue where they are coming from:

2024/05/23 07:46:47 [ERROR] [2024-05-23 07:46:47,808 __main__ ERROR] Unexpected error
2024/05/23 07:46:47 [ERROR] [2024-05-23 07:46:47,809 __main__ ERROR] Error applying config, will try again in 1 seconds
2024/05/23 07:47:27 [ERROR] [2024-05-23 07:47:27,097 __main__ ERROR] Unexpected error
2024/05/23 07:47:27 [ERROR] [2024-05-23 07:47:27,098 __main__ ERROR] Error applying config, will try again in 1 seconds
2024/05/23 07:47:35 [ERROR] [2024-05-23 07:47:35,095 __main__ ERROR] Unexpected error
2024/05/23 07:47:35 [ERROR] [2024-05-23 07:47:35,095 __main__ ERROR] Error applying config, will try again in 2 seconds

What additional information you need for the above errors?

--- a/docs/RELEASE-NOTES.rst
+++ b/docs/RELEASE-NOTES.rst
@@ -15,6 +15,7 @@ Bug Fixes
 ````````````
 * `Issue 3401 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3401>`_: Fix for invalid iRule generation for HTTP/2 full proxy mode
 * `Issue 3466 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3466>`_: Fix scaling issue in NodeportLocal mode
+* `Issue 3432 <https://github.com/F5Networks/k8s-bigip-ctlr/issues/3432>`_: Show meaningful logs for exceptions occured from controller agent
 
 Upgrade notes
 ``````````````

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
 -e git+https://github.com/f5devcentral/f5-cccl.git@7f394eb53d6d777e5d057d9ef5fe2ba01ce6d1cd#egg=f5-cccl
--e git+https://github.com/f5devcentral/f5-ctlr-agent.git@844e9cb1fb4612713761d622a7566b860775c0b5#egg=f5-ctlr-agent
+-e git+https://github.com/f5devcentral/f5-ctlr-agent.git@47421dc9f80b611197195dac86344dde3d46f610#egg=f5-ctlr-agent
 -e git+https://github.com/F5Networks/f5-icontrol-rest-python.git@3fee4a4599e903cce1abfe232e6ffb74d7085b64#egg=f5-icontrol-rest

